src/models/account.cpp

```diff
@@ -507,6 +529,181 @@ int Account::importFromCSV(const std::string& path)
     return imported;
 }
 
+int Account::importFromOFX(const std::string& path)
+{
+    int imported{ 0 };
+    unsigned int nextId{ getNextAvailableTransactionId() };
+    std::regex xmlTagRegex{ "^<([^/>]+|/STMTTRN)>(?:[+-])?([^<]+)" };
+    std::smatch xmlTagMatch;
+    Transaction* transaction{ nullptr };
```

Comment:
Let's switch to `std::shared_ptr`

The line should now read
```cpp
std::shared_ptr<Transaction> transaction;
```
It will automatically be initialized to `nullptr` by default.